src/config.rs

```diff
@@ -2771,6 +2778,31 @@ mod tests {
         assert!(tikv_cfg.check_critical_cfg_with(&last_cfg).is_ok());
     }
 
+    #[test]
+    fn test_last_cfg_modified() {
+        let mut tikv_cfg = TiKvConfig::default();
```

Comment:
Should use `let (mut cfg, tmp_dir) = TiKvConfig::with_tmp().unwrap()` to avoid leaving config files after runing test.

Reply:
Thanks!